mlflow/store/tracking/rest_store.py

```diff
@@ -535,7 +536,25 @@ def log_batch(self, run_id, metrics, params, tags):
         self._call_endpoint(LogBatch, req_body)
 
     def record_logged_model(self, run_id, mlflow_model):
-        req_body = message_to_json(LogModel(run_id=run_id, model_json=mlflow_model.to_json()))
+        ## Ignore config while logging model
+        model_info = {
+            key: {
+                flavor_name: {
+                    flavor_key: flavor_value
+                    for flavor_key, flavor_value in flavor_dict.items()
+                    if flavor_key != "config"
+                }
+                if "config" in flavor_dict
+                else flavor_dict
+                for flavor_name, flavor_dict in value.items()
+            }
+            if key == "flavors"
+            else value
+            for key, value in mlflow_model.to_dict().items()
+        }
```

Comment:
according to @prithvikannan 's suggestion, we only need to log the 3 field in `mlflow_model`:

`flavors`, `artifact_path`, `utc_time_created`

Reply:
we also have file store / Sql alchemy store backend, could you update them too ?

Reply:
config is part of `flavors` and the error is caused because of the size of config. Thats why I just removed config from flavors

Reply:
@WeichenXu123 to simplify I have added `with_config` flag where we can remove the config when we do `to_dict` or `to_json` and that flag will be set at false for all 3 stores when running `record_logged_model`

Reply:
got it, excluding `config` looks good to me. CC @prithvikannan